packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -137,7 +137,7 @@ export class WaferMap extends FoundationElement {
      */
     @observable public hoverDie: WaferMapDie | undefined;
 
-    @observable public highlightedValues: string[] = [];
+    @observable public highlightedTags: string[] = [];
```

Comment:
While it looks like this is the agreed upon terminology for this API in the discussion on the PR for the HLD, the spec document itself was not updated to reflect this. Can you correct that?